### PR TITLE
README: badges and more accurate description

### PR DESCRIPTION
**Pull request by ancorgs**
For a preview, see https://github.com/ancorgs/yast-storage-ng/tree/readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,29 @@
...
 
-  * MD RAIDs can be changed to be partitioned or unpartitions on the fly
+The module is **entirely unsupported**.
 
-- use of libstorage-ng from Ruby/YaST without legacy code
+## Installation
 
-The module is entirely unsupported.
+If you are brave enough to install this module in a (open)SUSE system, despite
+the facts that is not supported and can produce data loss, you can perform the
```
Comment (lslezak) [inline README.md]: NTH: I'd put `is not supported and can produce data loss` in bold so it is more prominent.

File: README.md [hunk 2]
```diff
@@ -1,13 +1,29 @@
...
 
-This is a technoligy preview to demonstrate:
+[![Build Status](https://travis-ci.org/yast/yast-storage-ng.svg?branch=master)](https://travis-ci.org/yast/yast-storage-ng)
+[![Coverage
+Status](https://coveralls.io/repos/github/yast/yast-storage-ng/badge.svg?branch=master)](https://coveralls.io/github/yast/yast-storage-ng?branch=master)
+[![Code
+Climate](https://codeclimate.com/github/yast/yast-storage-ng/badges/gpa.svg)](https://codeclimate.com/github/yast/yast-storage-ng)
 
-- libstorage-ng supports:
+yast2-storage-ng is a reimplementation of the YaST storage library (yast2-storage)
```
Comment (lslezak) [inline README.md]: NP: `storage library` is not accurate as there is also frontend (clients), probably `storage module` it a bit better.

**Comment by lslezak** [comment]
LGTM

File: README.md (hunk 1 above)
Comment (ancorgs) [inline README.md]: Ok, but then I will remove the bold "entirely unsupported" from the previous paragraph. Let's keep the calm. :wink:

**Comment by lslezak** [comment]
LGTM, thanks!

**Pull request merged**